torch/ao/quantization/fx/prepare.py

```diff
@@ -774,10 +766,23 @@ def _recursive_maybe_replace_node_with_obs(
 
     new_args = []
     for old_arg in graph_output_node.args:
-        new_args.append(
-            _recursive_maybe_replace_node_with_obs(
-                old_arg, output_target_dtype, node_name_to_target_dtype,
-                qconfig_map, model, modules, graph))
+        if isinstance(old_arg, (list, tuple)):
```

Comment:
what if we return a `dict`, what does it look like?

Reply:
do we need to support dict as well? 
list and tuple are common outputs, i didn't see the use case for output the dict of tensor.

Reply:
even if we don't support it, I feel we should test and maybe add an assert to be clear

Reply:
would be good to add a dict output test case as well

Reply:
Sure will add it

Reply:
btw, we do have dict output use cases in d2go, I'm adding a fix in convert right now